.github/workflows/lwcE2E.yml

```
@@ -0,0 +1,139 @@
+name: LWC End to End Tests
+
+on:
+  workflow_dispatch:
+    inputs:
+      automationBranch:
+        description: 'Set the branch to use for automation tests'
+        required: false
+        default: 'develop'
+        type: string
+      lwcLsp:
+        description: 'LWC LSP'
+        required: false
+        default: true
+        type: boolean
+      debugLwcTests:
+        description: 'Debug LWC Tests'
+        required: false
+        default: true
+        type: boolean
+      runLwcTests:
+        description: 'Run LWC Tests'
+        required: false
+        default: true
+        type: boolean
+      vscodeVersion:
+        description: 'VSCode Version'
+        required: false
+        default: '1.85.2'
+        type: string
+      runId:
+        description: 'Run ID of the workflow run that created the vsixes'
+        required: true
+        type: string
+
+  workflow_call:
+    inputs:
+      automationBranch:
+        description: 'Set the branch to use for automation tests'
+        required: false
+        default: 'develop'
+        type: string
+      lwcLsp:
+        description: 'LWC LSP'
+        required: false
+        default: true
+        type: boolean
+      debugLwcTests:
+        description: 'Debug LWC Tests'
+        required: false
+        default: true
+        type: boolean
+      runLwcTests:
+        description: 'Run LWC Tests'
+        required: false
+        default: true
+        type: boolean
+      vscodeVersion:
+        description: 'VSCode Version'
+        required: false
+        default: '1.85.2'
+        type: string
+      runId:
+        description: 'Run ID of the workflow run that created the vsixes'
+        required: true
+        type: string
+
+jobs:
+  lwcLsp:
+    if: ${{ inputs.lwcLsp }}
+    uses: ./.github/workflows/runE2ETest.yml
+    secrets: inherit
+    with:
+      automationBranch: ${{ inputs.automationBranch }}
+      testToRun: 'lwcLsp.e2e.ts'
+      vscodeVersion: ${{ inputs.vscodeVersion || '1.85.2' }}
+      runId: ${{ inputs.runId }}
+
+  debugLwcTests:
+    if: ${{ inputs.debugLwcTests }}
+    uses: ./.github/workflows/runE2ETest.yml
+    secrets: inherit
+    with:
+      automationBranch: ${{ inputs.automationBranch }}
+      testToRun: 'debugLwcTests.e2e.ts'
+      vscodeVersion: ${{ inputs.vscodeVersion || '1.85.2' }}
+      runId: ${{ inputs.runId }}
+
+  runLwcTests:
+    if: ${{ inputs.runLwcTests }}
+    uses: ./.github/workflows/runE2ETest.yml
+    secrets: inherit
+    with:
+      automationBranch: ${{ inputs.automationBranch }}
+      testToRun: 'runLwcTests.e2e.ts'
+      vscodeVersion: ${{ inputs.vscodeVersion || '1.85.2' }}
+      runId: ${{ inputs.runId }}
+
+  slack_success_notification:
+    if: ${{ success() }}
+    needs: [lwcLsp, debugLwcTests, runLwcTests]
+    uses: ./.github/workflows/slackNotification.yml
+    secrets: inherit
+    with:
+      title: 'Lwc E2E Tests'
```

🙀 I missed this one!  (Title is present in all 3 Slack notifications)

```suggestion
      title: 'LWC E2E Tests'
```